sdk/servicebus/service-bus/test/internal/batchReceiver.spec.ts

```diff
@@ -1144,7 +1134,7 @@ describe("Batching Receiver", () => {
     });
   });
 
-  describe(withSessionTestClientType + ": Batch Receiver - disconnects", function(): void {
+  describe.only(withSessionTestClientType + ": Batch Receiver - disconnects", function(): void {
```

Comment:
Careful now...

Reply:
One of the test in this suite is failing in the pipeline only on windows but not on local.. Investigating with this.